packages/@expo/prebuild-config/src/plugins/unversioned/expo-splash-screen/withAndroidSplashImages.ts

```diff
@@ -151,8 +151,11 @@ async function clearAllExistingSplashImagesAsync(projectRoot: string) {
     Object.values(DRAWABLES_CONFIGS).map(async ({ modes }) => {
       await Promise.all(
         Object.values(modes).map(async ({ path: filePath }) => {
-          if (await fs.pathExists(path.resolve(androidMainPath, filePath))) {
-            await fs.remove(path.resolve(androidMainPath, filePath));
+          if (fs.existsSync(path.resolve(androidMainPath, filePath))) {
+            await fs.promises.rm(path.resolve(androidMainPath, filePath), {
+              force: true,
+              recursive: true,
+            });
           }
```

Comment:
```suggestion
          await fs.promises.rm(path.resolve(androidMainPath, filePath), {
            force: true,
            recursive: true,
          });
```

nit: since `force: true` maybe not necessarily to check exists

Reply:
Ah, so I checked and I thought that Node.js mirrors POSIX `rm` behaviour and would stop on write-protected files, but it indeed does not, so we can safely drop this 👍